assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationIntroductoryOverlayNotification.js

```diff
@@ -58,6 +61,20 @@ export default function AudienceSegmentationIntroductoryOverlayNotification() {
 		);
 	};
 
+	const scrollToWidgetAndDismissNotification = ( event ) => {
+		event.preventDefault();
+
+		const widgetClass =
+			'.googlesitekit-widget-area--mainDashboardTrafficPrimary';
```

Comment:
I realised this is as per the IB, but in fact we should scroll to the audience segmentation widget area:

```suggestion
		const widgetAreaClass =
			'.googlesitekit-widget-area--mainDashboardTrafficAudienceSegmentation';
```